### DISCUSSION TITLE
Add "count" property in GraphQL API for collections

### DISCUSSION THREAD

**Discussion by KonradHEXANT** [directus/directus#3881] [Feature Requests]
It would be very useful to know how many items each collection has. This information would be great to control for example: the "load more" button if visible items qty is lower than count etc...

**Comment by benhaynes** [comment]
Our REST API delivers this in the metadata (I believe), so I agree that this info should be available within GraphQL to reach API feature parity. 👍

**Comment by rijkvanzanten** [comment]
Aggregation capabilities have since been implemented in GraphQL as well as REST 🙂
